includes/flattened_custom_event.js

```diff
@@ -0,0 +1,152 @@
+const getKeys = (keys) => {
+    return keys.map(key => ({
+        [key.alias || key.keyName]: {
+          description: key.description,

```

Comment:
AK to check: Is this alias if it exists, keyName if alias doesn't exist?

Reply:
It's because null is a 'false-ey' value so the OR operator || does not return it and returns the object on its right instead.

See https://www.geeksforgeeks.org/how-to-replace-a-value-if-null-or-undefined-in-javascript/

Reply:
Checked.